sparql/sparql11/protocol/manifest.ttl

```diff
@@ -1,961 +1,1430 @@
+                ht:elementName "application/sparql-update";
+                ht:params ([
+                    a ht:Parameter;
+                    ht:paramName "charset";
+                    ht:paramValue "UTF-16"
+                  ])
+              ])
+          ]);
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "4XX"
+        ]
+      ])
+  ];
+  mf:name "invoke update operation with direct POST, but with a non-UTF8 encoding" .
+
+<#bad_update_syntax> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql?using-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "update=CLEAR%20XYZ"
+        ];
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "4XX"
+        ]
+      ])
+  ];
+  mf:name "invoke update operation with invalid update syntax (4XX result)" .
+
+<#bad_update_wrong_media_type> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql?using-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "CLEAR NAMED"
+        ];
+        ht:headers ([
+            a ht:RequestHeader;
+            ht:fieldName "Content-Type";
+            ht:fieldValue "text/plain";
+            ht:headerElements ([
+                a ht:HeaderElement;
+                ht:elementName "text/plain"
+              ])
+          ]);
+        ht:httpVersion "1.1";
+        ht:methodName "POST";
+        ht:resp [
+          a ht:Response;
+          ht:statusCodeValue "4XX"
+        ]
+      ])
+  ];
+  mf:name "invoke update operation with a POST with media type that's not url-encoded or application/sparql-update" .
+
+<#query_content_type_ask> a mf:ProtocolTest;
+  dawg:approval dawg:Approved;
+  dawg:approvedBy <http://www.w3.org/2009/sparql/meeting/2012-11-20#resolution_3>;
+  mf:action [
+    a ht:Connection;
+    ht:connectionAuthority "www.example";
+    ht:requests ([
+        a ht:Request;
+        ht:absolutePath "/sparql";
+        ht:body [
+          a cnt:ContentAsText;
+          cnt:characterEncoding "UTF-8";
+          cnt:chars "query=ASK%20%7B%7D&default-graph-uri=http%3A%2F%2Fkasei.us%2F2009%2F09%2Fsparql%2Fdata%2Fdata0.rdf"
```

Comment:
Original test does not specify dataset, and specifies query via direct-POST request body. Missing `Content-Type: application/sparql-query` header.